modules/common/src/main/scala/com/snowplowanalytics/snowplow/storage/bigquery/common/metrics/StdoutReporter.scala

```diff
@@ -0,0 +1,44 @@
+/*
+ * Copyright (c) 2018-2021 Snowplow Analytics Ltd. All rights reserved.
+ *
+ * This program is licensed to you under the Apache License Version 2.0,
+ * and you may not use this file except in compliance with the Apache License Version 2.0.
+ * You may obtain a copy of the Apache License Version 2.0 at http://www.apache.org/licenses/LICENSE-2.0.
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the Apache License Version 2.0 is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the Apache License Version 2.0 for the specific language governing permissions and limitations there under.
+ */
+package com.snowplowanalytics.snowplow.storage.bigquery.common.metrics
+
+import cats.effect.{Resource, Sync}
+
+import org.typelevel.log4cats.Logger
+
+import com.snowplowanalytics.snowplow.storage.bigquery.common.config.model.Monitoring.Stdout
+
+object StdoutReporter {
+
+  def make[F[_]: Logger: Sync](config: Stdout): Resource[F, Metrics.Reporter[F]] =
+    Resource.eval[F, Metrics.Reporter[F]](Sync[F].delay(new Metrics.Reporter[F] {
+      def report(snapshot: Metrics.MetricsSnapshot): F[Unit] =
+        snapshot match {
+          case lms: Metrics.MetricsSnapshot.LoaderMetricsSnapshot =>
+            val totalEventCount = lms.goodCount + lms.badCount + lms.failedInsertCount
+            val logStart = s"${Metrics.normalizeMetric(config.prefix, "StatisticsPeriod")} = ${config.period}"
```

Comment:
Since I saw `prefix` usage on enrich and something similar is also used on `StatsDReporter`, I used prefix here too. But do you think there is any use case for it ? When I look at the logs, it seems to me it is cluttering the logs a bit.

Reply:
Personally, I'm not so worried about the prefix, but do you think `logPeriod` might be a better name for it? Because it's not strictly when logging started but what period it covers.

Reply:
I guess I've used `start` wrongly here. It is should be more like a `logHead` because what I want to imply is this is the beginning of the log line. But `logPeriod` also sounds good.